### ISSUE TITLE
Some emoji (e.g. ‼️) are rendered differently in editor vs. Preview

### ISSUE THREAD

**Issue by borekb** [notaapp/talk#229]
For example, the ‼️ emoji renders like this in the editor:

<img width="108" alt="Screen Shot 2022-05-23 at 17 36 25-shadow" src="https://user-images.githubusercontent.com/101152/169856296-b16744f0-19c7-4836-96e8-dc5d83edfd65.png">

but like this in Preview:

<img width="108" alt="Screen Shot 2022-05-23 at 17 36 39-shadow" src="https://user-images.githubusercontent.com/101152/169856174-c9447b00-b59a-44f0-bbc9-a7f54be0caee.png">

Is it because of a different font used or something? I know this is happening in other web apps as well, not sure if there's something Nota can do about it or not.

**Comment by borekb** [comment]
Here in GitHub, it's the opposite: double exclamation mark ‼️ is "textual" when in editor but a full emoji when rendered 🤷‍♂️.

<img width="941" alt="Screen Shot 2022-05-23 at 17 41 06" src="https://user-images.githubusercontent.com/101152/169856688-e4b534d4-f9a8-483a-98dc-b0958a8d0485.png">

<img width="941" alt="Screen Shot 2022-05-23 at 17 41 12" src="https://user-images.githubusercontent.com/101152/169856707-cbed09b5-ee53-4a5a-b00a-38d5e480b433.png">

**Comment by astoilkov** [comment]
Yes. It's because of the font. We can change that. It's a good idea to make it consistent with the editor. Thanks.
